client/state/form/actions.js

```diff
@@ -7,3 +14,68 @@ export const setFormProperty = ( field, value ) => {
 		value,
 	};
 };
+
+export const goToStep = ( stepIndex ) => {
+	return {
+		type: GO_TO_STEP,
+		step: stepIndex,
+	};
+};
+
+export const nextStep = () => ( dispatch, getState, { callbackURL, nonce, formSchema, formLayout } ) => {
```

Comment:
The "extra" argument, in this case an object with the properties `{ callbackURL, nonce, formSchema, formLayout }` is configured when initializing the `thunk` middleware (`main.js`, line 22) and so they are global and immutable.